src/api/app/controllers/statistics/maintenance_statistics_controller.rb

```diff
@@ -1,12 +1,24 @@
 module Statistics
   class MaintenanceStatisticsController < ApplicationController
-    skip_before_action :extract_user
+    skip_before_action :extract_user, :require_login
```

Comment:
I can move the public routing there, if you want. Just note that we already route calls to public/* to the respective controllers.